docs/_docs/hivemq/schemas.md

```diff
@@ -125,6 +125,69 @@ $ mqtt hivemq schemas get --id my-schema-id
 
 {% include options/help-options.md defaultHelp=true %}
 
+***
+
+# List Schemas
+
+*** 
+
+List every schema that exists on a HiveMQ node.
+This list may optionally be filtered by schema id and schema type.
+
+```
+mqtt hivemq schemas list
+```
+
+***
+
+## Simple Example
+
+```
+$ mqtt hivemq schemas list --id schema-1
+{
+  "items": [
+    {
+      "id": "schema-1",
+      "version": 1,
+      "createdAt": "2023-06-12T00:38:34.911Z",
+      "type": "JSON",
+      "schemaDefinition": "ewogICJkZXNjcmlwdGlvbiI6ICJUaGlzIGlzIGFub3RoZXIgZ2VuZXJpYyBKU09OIHNjaGVtYSwgc2luY2UgaXQgcmVxdWlyZXMganVzdCBhIEpTT04sIG5vdGhpbmcgZnVydGhlciBzcGVjaWZpZWQiLAogICJ0eXBlIjogIm9iamVjdCIKfQo",
+      "arguments": {}
+    },
+    {
+      "id": "schema-1",
+      "version": 2,
+      "createdAt": "2023-06-12T09:57:55.862Z",
+      "type": "JSON",
+      "schemaDefinition": "ewogICJkZXNjcmlwdGlvbiI6ICJUaGlzIGlzIGEgdGhlIG1vc3QgZ2VuZXJpYyBKU09OIHNjaGVtYSwgc2luY2UgaXQgcmVxdWlyZXMganVzdCBhIEpTT04sIG5vdGhpbmcgZnVydGhlciBzcGVjaWZpZWQiLAogICJ0eXBlIjogIm9iamVjdCIKfQ==",
+      "arguments": {}
+    }
+  ]
+}
+```
+
+***
+
+## Options
+
+| Option | Long Version | Explanation                                                                   | Required |
+|--------|--------------|-------------------------------------------------------------------------------|:--------:|
+| `-i`   | `--id`       | Filter by schemas with a matching schema id. Can be specified multiple times. |          |
+| `-t`   | `--type`     | Filter by schemas of a matching schema type. Can be specified multiple times  |          |
```

Comment:
This automatically fetches all pages, so `limit` would have no effect. Do you think it would make sense to offer a separate mode that returns one page at a time?

`fields` isn't supported because we decided it would not have much use. You can just use `jq` or something to filter the result.

Reply:
Yes, I thought about a "total-limit" option.

Hmm, fields could spare some bandwidth as the fields are not sent by the server in this case and make the total transmission time shorter.